nifi-commons/nifi-jetty-configuration/src/main/java/org/apache/nifi/jetty/configuration/connector/StandardServerConnectorFactory.java

```diff
@@ -68,6 +68,8 @@ public class StandardServerConnectorFactory implements ServerConnectorFactory {
 
     private String[] includeSecurityProtocols = INCLUDE_ALL_SECURITY_PROTOCOLS;
 
+    private int maxRequestHeaderSize = 8192;
```

Comment:
Recommend renaming this to `requestHeaderSize` to align with the Jetty HttpConfiguration property name.

Reply:
I renamed this variable and the method name in this class to align with Jetty HttpConfiguration.